microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md

```diff
@@ -0,0 +1,120 @@
+---
```

Comment:
@vchapel - https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/#detached-elements
typo that could reasonably be fixed using this PR.  It's in a different .md file than the new .md file, but is on-topic for this PR:
Memory leaks in web applications are a difficult to locate and repair.